README.md

```diff
@@ -15,6 +15,29 @@ To install this module type the following:
    make test
    make install
 ```
+To install debian jessie dependences:
+```
+Install enyevent
+   sudo apt-get install libanyevent-perl
+   sudo cpan AnyEvent::MPRPC::Client
+   
+Install moose
+   sudo apt-get instal libmoose-perl
```

Comment:
i would just make the apt-get install stuff on one line. also use App-cpanminus to install modules instead of `sudo cpan`. cpanminus > cpan